docs/src/test-global-setup-teardown-js.md

```diff
@@ -26,25 +45,184 @@ export default defineConfig({
       dependencies: ['setup'],
     },
     {
-      name: 'firefox',
-      use: { ...devices['Desktop Firefox'] },
-      dependencies: ['setup'],
+      // other project
+    }
+  ]
+});
+```
+### Setup Example
+
+This example will show you how to use project dependencies to create a global setup that logs into an application and saves the state in storage state. This is useful if you want to run multiple tests that require a sign in state and you want to avoid logging in for each test.
```

Comment:
```suggestion
This example will show you how to use project dependencies to create a global setup that logins into an application and saves the state in storage state. This is useful if you want to run multiple tests that require a sign sign-in state and you want to avoid login for each test.
```